lib/tasks/cron/5-past-half-hour.rake

```diff
@@ -2,7 +2,8 @@ namespace :cron do
   task '5-past-half-hour': :log_to_stdout do
     Rails.logger.debug 'Starting 5-past-half-hour cron'
 
-    Rails.logger.info "UpdateUserContactInfo.call"
+    Rails.logger.info "Starting Salesforce Contact and School Sync"
+    OpenStax::RescueFrom.this { UpdateSchoolSalesforceInfo.call }
     OpenStax::RescueFrom.this { UpdateUserContactInfo.call }
```

Comment:
It may be a good idea to check how long these take in prod actually, to make sure combined they don't exceed 30 min

Reply:
Sometimes they do - does that mean they are going to start running again? I thought we had something that would prevent duplicate crons from starting if the previous wasn't complete.

Reply:
Oh, you know, I think I fixed what was causing these to run so long. Looks like they are down to sub-minute runs now. Probably good.
https://openstax.sentry.io/crons/accounts/update-user-contact-info/?environment=production

Reply:
yeah, and it looks like it's handling whatever salesforce throws at it through active_force https://github.com/openstax/active_force/blob/3210375b8ea95b576c876de87f1e02fcdc0139d0/lib/active_force/mapping.rb#L63